src/nvim/ex_docmd.c

```diff
@@ -4184,9 +4184,6 @@ static char_u *invalid_range(exarg_T *eap)
         }
         break;
       case ADDR_UNSIGNED:
-        if (eap->line2 < 0) {
-          return (char_u *)_(e_invrange);
-        }
         break;
```

Comment:
Delete this line if you're doing this. `ADDR_NONE` and `ADDR_UNSIGNED`  cases are grouped in other switch/case code.

Reply:
I assume you mean the line break line? OK, I've removed it. Thanks for the help.